pyanaconda/modules/network/nm_client.py

```diff
@@ -58,12 +58,16 @@ def get_iface_from_hwaddr(nm_client, hwaddr):
                                         NM.DeviceType.WIFI):
             try:
                 address = device.get_permanent_hw_address()
+                if not address:
+                    address = device.get_hw_address()
```

Comment:
I'd maybe keep only get_pernmanent_hw_address in the try: block and assign None to address in the except:, then using get_hw_address() if address is None, but I think it is OK this way as well.